snapshot/store/groupcache_store.go

```diff
@@ -65,6 +65,17 @@ func MakeGroupcacheStore(underlying Store, cfg *GroupcacheConfig, ttlc *TTLConfi
 			}
 			return ttl, dest.SetBytes(data)
 		}),
+		groupcache.ContainerFunc(func(ctx groupcache.Context, bundleName string) (bool, error) {
+			log.Info("Not cached, try to check bundle existence and populate cache metadata: ", bundleName)
```

Comment:
log msg - doesn't try to populate cache metadata

Reply:
Will fix